app/helpers/casino/sessions_helper.rb

```diff
@@ -23,6 +23,10 @@ def current_user
     tgt.user
   end
 
+  def current_authenticator_context
+    CASino.config.authenticator_context_builder.call(params, request)
```

Comment:
What is the suggested approach to set `CASino.config.authenticator_context_builder`?